config/src/net.rs

```diff
@@ -87,6 +87,13 @@ impl FromStr for Address {
     type Err = Error;
 
     fn from_str(addr: &str) -> Result<Self, Error> {
+        // unix abstract socket address, `man 7 unix` for system descriptions
+        if addr.starts_with("unix://@") {
+            return Ok(Self::Unix {
+                path: addr.trim_start_matches("unix://").to_owned(),
+            });
+        }
```

Comment:
It might make sense to file this logic below with the rest of the code that handles the `unix://` prefix.

```suggestion
        if let Some(path) = addr.strip_prefix("unix://@") {
            return Ok(Self::Unix {
                path: path.to_owned(),
            });
        }
```

Reply:
yes, the '@' should be kept, your suggestions can not run.

__'moving this code block to the rest of the code that handles the `unix://` prefix'__, this is a good idea, but a `unix://@...` is very likely to be denied by the following `Url::parse`

Reply:
the change I commit are the only one case that get tested in my practice